frappe/email/doctype/email_queue/email_queue.py

```diff
@@ -259,6 +259,14 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 				)
 			else:
 				update_fields.update({"status": "Error"})
+				notification = frappe.new_doc("Notification Log")
+				notification.for_user = self.queue_doc.owner
+				notification.set("type", "Alert")
+				notification.from_user = self.queue_doc.owner
+				notification.document_type = self.queue_doc.doctype
+				notification.document_name = self.queue_doc.name
+				notification.subject = f"Failed to send email for {self.queue_doc.reference_doctype}({self.queue_doc.reference_name})"
```

Comment:
Let's use email subject instead. That gives better context for which email failed. 

Also, you should use translations. https://frappeframework.com/docs/user/en/translations

```suggestion
				notification.subject = _("Failed to send email for") + f" {self.queue_doc.reference_doctype}({self.queue_doc.reference_name})"
```